doc/source/advanced.rst

```diff
@@ -376,3 +376,14 @@ No continuation
 
 
 Note, that this only forces start of a new large file - it is still possible to continue the process with either auto or manual modes.
+
+
+****************************
+SHA-1 hashes for large files
+****************************
+
+Depending the number and size of sources and the size of the result file, the SDK may decide to use the large file API to create a file on the server.  In such cases the file's SHA-1 won't be stored on the server and will not be returned in the ``X-Bz-Content-Sha1`` header.  The SHA-1 can optionally still be stored with the file in the ``large_file_sha1`` entry in the ``file_info``.
+
+In basic scenarios, large files uploaded to the server will have a ``large_file_sha1`` element added automatically to their ``file_info``.  However, when concatenating multiple sources, it may be impossible for the SDK to figure out the SHA-1 automatically.  In such cases, the SHA-1 can be provided using the ``large_file_sha1`` parameter to :meth:`b2sdk.v2.Bucket.create_file`, :meth:`b2sdk.v2.Bucket.concatenate` and their stream equivalents.  If the parameter is skipped or ``None``, the result file may not have the ``large_file_sha1`` value set.
+
+Note that the provided SHA-1 value is not verified.
```

Comment:
does this mean it's also not used to validate that data has not been corrupted when uploading?

Reply:
yes. The server doesn't support that - different parts land in different places and can be uploaded out of order so it's practically impossible for the server to calculate a hash of a large file